src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsId;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public List<Long> getTicketsId() {
+        return ticketsId;
+    }
+
+    public void setTicketsId(List<Long> ticketsId) {
```

Comment:
Consider renaming the method 'setTicketsId(List<Long> ticketsId)' to 'setTicketIds(List<Long> ticketIds)' to match the field name change.